cmake/therock_subproject.cmake

```diff
@@ -1003,8 +1031,8 @@ function(_therock_cmake_subproject_setup_toolchain
   endif()
 
   # Configure additional HIP dependencies.
-  if (compiler_toolchain STREQUAL "amd-hip")
-  _therock_assert_is_cmake_subproject("hip-clr")
+  if (compiler_toolchain STREQUAL "amd-hip" AND NOT WIN32)
```

Comment:
This is a pretty big logic change. I'm open to handling it incrementally, but at least a comment please.

Reply:
How does this look now? I've added this new code instead of changes to two conditionals:

```cmake
    if(WIN32 AND compiler_toolchain STREQUAL "amd-hip")
      # On Windows, we only have the amd-llvm toolchain today, so override.
      set(compiler_toolchain "amd-llvm")
      if(THEROCK_VERBOSE)
        message(STATUS "Windows: overriding compiler_toolchain from amd-hip to amd-llvm")
      endif()
    endif()
```

Reply:
Lgtm. Just looking for some commentary to help future us.